create_fluent_bit_config.py

```diff
@@ -0,0 +1,273 @@
+import os
+
+# Function to create the Fluent Bit configuration
+def create_fluent_bit_config():
+    # Get log level from environment variable
+    log_level = os.getenv('LOG_LEVEL', 'info')
+
+    # Base configuration for Fluent Bit SERVICE and INPUT sections
+    config = f"""
+[SERVICE]
+    Parsers_File parsers.conf
+    Parsers_File parsers_multiline.conf
+    Flush        1
+    Daemon       Off
+    Log_Level    {log_level}
+"""
+
+    # Add multiline parser configuration if specified
+    if os.getenv('MULTILINE_START_STATE_RULE'):
+        config += f"""
+[INPUT]
+    Name         tail
+    Path         /var/lib/docker/containers/*/*.log
+    Parser       docker
+    Tag          docker.*
+    read_from_head {os.getenv('READ_FROM_HEAD', 'true')}
+    multiline.parser multiline-regex
+"""
```

Comment:
suggestion, maybe instead of adding the full config here and other places in this function, have helper functions per config block, for example `_get_input_multiline()`, `_get_input_no_multiline()`... I think the code will be easier to read that way

Reply:
I Adjusted the code to generate the Fluent Bit configuration by combining config blocks